.github/workflows/tests.yml

```
@@ -233,6 +235,17 @@ jobs:
         with:
           jobs: ${{ toJSON(needs) }}
 
+      - name: Report failures
+        if: github.event_name == 'schedule' && failure()
+        uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I think we should use the conda-bot token for that, if you don't mind? 
```suggestion
          GITHUB_TOKEN: ${{ secrets.CONDA_LIBMAMBA_SOLVER_ISSUES }}
```